admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -2,38 +2,215 @@
 Configuration Parameters
 ========================
 
-Nextcloud uses the ``config/config.php`` file to control server operations.
-``config/config.sample.php`` lists all the configurable parameters within
-Nextcloud, along with example or default values. This document provides a more
-detailed reference. Most options are configurable on your Admin page, so it
-is usually not necessary to edit ``config/config.php``.
-
-.. note:: The installer creates a configuration containing the essential parameters.
-   Only manually add configuration parameters to ``config/config.php`` if you need to
-   use a special value for a parameter. **Do not copy everything from**
-   ``config/config.sample.php`` **. Only enter the parameters you wish to modify!**
-
-Multiple config.php file
-------------------------
+Introduction
+------------
+
+Nextcloud uses ``config/config.php`` as its main configuration file. This file controls 
+various fundamental aspects of server operations. It is typically modified as part of initial 
+deployment, when troubleshooting, and when making adjustments to surrounding infrastructure.
+
+This is a required file for all Nextcloud deployments and thus it is critical for Nextcloud 
+administrators to be familiar with managing it.
+
+This section of the *Administration Manual* documents how to adjust this essential file, 
+certain special characteristics of the ``config/`` directory, and all of the supported 
+parameters that can be specified in a ``config/config.php`` file.
+
+.. note:: While ``config/config.php`` is a required file, many Nextcloud or Nextcloud app
+   settings are managed elsewhere and thus not included in it. These settings are typically
+   managed via individual apps.
+
+Loading
+-------
+
+Configuration files located in ``config/`` are parsed automatically when Nextcloud 
+starts up. They are also checked for changes periodically (approximately every two seconds 
+in a standard PHP environment running with default *OPcache* settings; approximately every 
+sixty seconds in many pre-packaged Nextcloud installation methods).
+
+The ``config/config.php`` file may be supplemented by additional ``*.config.php`` files 
+placed in the ``config/`` directory (if appropriately named and formatted).
+
+.. danger:: Be cautious when naming or creating backup copies of your active 
+   ``config/config.php``. If a backup is located within ``config/`` and is named
+   ``(ANYTHING).config.php``, it will be loaded as part of your live configuration
+   and override your ``config/config.php`` values!
+
+.. tip:: If your configuration changes don't seem to be taking effect, check: (a) your PHP opcache 
+   configuration; (b) for additional ``*.config.php`` files located in ``config/``; (c) the documentation
+   for your Nextcloud installation method/package; (d) the output of ``occ config:list system``.
+
+Format
+------
+
+The short answer is that``config/`` files are plain text files with some special formatting 
```

Comment:
A space is missing after `that` and `\`\`config/\`\`` (I hope you get it, I don't manage to escape this in the comment correctly)

Reply:
Got it. Thanks, @tflidd!